build-logic/basics/src/main/kotlin/gradlebuild.repositories.gradle.kts

```diff
@@ -16,7 +16,20 @@
 
 repositories {
     maven {
-        name = "Gradle libs"
-        url = uri("https://repo.gradle.org/gradle/libs")
+        name = "Gradle public repository"
+        url = uri("https://repo.gradle.org/gradle/public")
+        content {
+            includeGroup("net.rubygrapefruit")
+            includeGroup("org.gradle.org.openjdk.jmc")
+            includeModule("flot", "flot")
+            includeModuleByRegex("org.gradle", "sample-(check|discovery)")
+        }
     }
+    google {
+        content {
+            includeGroup("com.android.databinding")
+            includeGroupByRegex("com\\.android\\.tools(\\.[a-z.\\-]*)?")
+        }
+    }
+    mavenCentral()
```

Comment:
I wonder if `mavenCentral()` should be first, so we resolve all we can from there.

Reply:
I think it's better to have a number of repositories that only declare include filters and a fallback repository for everything else. If we're doing it the other way around we'll be sending requests to maven central for stuff that we know is not available there.

Reply:
Sounds good, let's leave it like you have it.